src/humanize/time.py

```diff
@@ -386,7 +440,9 @@ def _suppress_lower_units(min_unit, suppress):
     return suppress
 
 
-def precisedelta(value, minimum_unit="seconds", suppress=(), format="%0.2f") -> str:
+def precisedelta(
+    value, minimum_unit="seconds", suppress=(), format="%0.2f", when=None
+) -> str:
```

Comment:
`precisedelta()` should not have `when` parameter. As discussed previously and explained in the newest docs for `naturaldelta()`, users can create their own `timedelta` inline if they need to do so from `datetime` objects. It does not make sense to have a `when` parameter since the delta functions return a time magnitude not a time vector.